nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-shell-authorizer/src/main/java/org/apache/nifi/authorization/ShellUserGroupProvider.java

```diff
@@ -272,7 +281,7 @@ public void onConfigured(AuthorizerConfigurationContext configurationContext) th
             }catch (final Throwable t) {
                 logger.error("", t);
             }
-        }, fixedDelay, fixedDelay, TimeUnit.SECONDS);
+        }, fixedDelay, fixedDelay, TimeUnit.MILLISECONDS);
```

Comment:
Why the change in units?

Reply:
The fixedDelay variable is being stored as milliseconds:

https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-shell-authorizer/src/main/java/org/apache/nifi/authorization/ShellUserGroupProvider.java#L322